axonserver-cli/src/main/java/io/axoniq/cli/CommandOptions.java

```diff
@@ -22,116 +22,227 @@
     /**
      * The base URL for Axon Server admin node.
      */
-    public static final Option ADDRESS = Option.builder("S").hasArg().longOpt("server").desc("Server to send command to (default http://localhost:8024)")
-                                               .build();
+    public static final Option ADDRESS = Option
+            .builder("S")
+            .longOpt("server")
+            .hasArg()
+            .desc("Server to send command to (default http://localhost:8024)")
+            .build();
+
+    /**
+     * Connect to Axon Server using HTTPS instead of HTTP.
+     */
+    public static final Option USE_HTTPS = Option
+            .builder("s")
```

Comment:
Seems like "s" is already taken for server

Reply:
Nope, that is capital-S, this is lowercase.